src/layout/index.tsx

```diff
@@ -51,30 +71,95 @@ const Layout: React.FC<LayoutProps> = ({
               </a>
             </Link>
           </h1>
-          <nav className={style.header__nav}>
-            <ul>
-              <li>
-                <Link href="/ansatte">
-                  <a>Alle varianter</a>
-                </Link>
-              </li>
-              <li>
-                <a href="https://jobs.variant.no" rel="noopener">
-                  Bli en variant
-                </a>
-              </li>
-              <li>
-                <a href="http://variant.blog" rel="noopener">
-                  Blogg
-                </a>
-              </li>
-              <li>
-                <a href="http://handbook.variant.no" rel="noopener">
-                  Håndbok
-                </a>
-              </li>
-            </ul>
-          </nav>
+
+          <span hidden id="menu-label">
+            Hovedmeny
+          </span>
+
+          <div ref={burgerRef}>
+            <button
+              className={style.container}
+              id="hamburger"
+              aria-labelledby="menu-label"
+              aria-expanded={clickActive ? true : false}
+              onClick={() =>
+                clickActive ? setClickActive(false) : setClickActive(true)
+              }
+            >
+              <div
+                className={and(
+                  style.bar1,
+                  clickActive ? style.bar1_change : '',
+                )}
+              />
+              <div
+                className={and(
+                  style.bar2,
+                  clickActive ? style.bar2_change : '',
+                )}
+              />
+              <div
+                className={and(
+                  style.bar3,
+                  clickActive ? style.bar3_change : '',
+                )}
+              />
+            </button>
+
+            <nav
+              className={and(
+                style.header__nav,
+                clickActive ? '' : style.header__nav__hidden,
+              )}
+              aria-labelledby="menu-label"
+              aria-hidden={clickActive ? true : false}
+              id="menu"
+            >
+              <ul className={style.header__nav__ul} id="nav-ul">
+                <li>
+                  <a
+                    href="https://jobs.variant.no"
+                    rel="noopener"
+                    tabIndex={clickActive ? 0 : -1}
```

Comment:
Hmm. What is this for as it is now? It causes the links to not be keyboard accessible on desktop.

Reply:
its there so that you cant tab to it when its not shown. Before i put that inn you could tab to the sidebar even if it was hidden to the side, it was accessible either way. When its -1 it disables tabbing to the links in the <ul.

Reply:
There are cases now on desktop where you cannot tab the visible menu in the header.

Reply:
I will look into that!

Reply:
Should be okey now, going to push soon. made a few changes